libsql/src/hrana/connection.rs

```diff
@@ -54,6 +54,10 @@ where
         self.current_stream().affected_row_count()
     }
 
+    pub fn total_changes(&self) -> u64 {
+        self.current_stream().total_changes()
```

Comment:
What happens when the stream changes on connection?
Hrana allows multiple streams on a single connection - how does that fit with this?

Reply:
ok I see that `total changes` comes from server and is stream independent, correct?

Reply:
Ok it actually does not come from the server. Shouldn't we accummulate the counter in connection not in stream?

Reply:
You are right, but it's a bit involved to fix it. I opened https://github.com/tursodatabase/libsql/issues/1512 for tracking this, but merging this as intermediate solution.